components/tidb_query/src/codec/mysql/time/mod.rs

```diff
@@ -468,6 +468,27 @@ mod parser {
             _ => None,
         }
     }
+
+    /// Try to parse a `u64` into a `Time` with given type and fsp
+    pub fn parse_from_u64(
+        ctx: &mut EvalContext,
+        input: u64,
+        time_type: TimeType,
+        fsp: u8,
+    ) -> Option<Time> {
+        let aligned = match input {
+            0 => 0,
```

Comment:
It's better to return `Some(Time::zero())` directly.

Reply:
OK